readthedocs/gold/urls.py

```diff
@@ -8,4 +9,10 @@
                        url(r'^edit/$', views.edit, name='gold_edit'),
                        url(r'^cancel/$', views.cancel, name='gold_cancel'),
                        url(r'^thanks/$', views.thanks, name='gold_thanks'),
+                       url(r'^projects/$', views.projects, name='gold_projects'),
+                       url(r'^projects/remove/(?P<project_slug>{project_slug})$'.format(
```

Comment:
All other URLs in the project have a trailing slash. We should keep it consistent.